pinot-common/src/main/java/org/apache/pinot/common/request/context/ExpressionContext.java

```diff
@@ -65,15 +65,6 @@ private ExpressionContext(Type type, String identifier, FunctionContext function
     _literal = literal;
   }
 
-  // TODO: Refactor all of the usage for getLiteralString.
-  @Deprecated
-  public String getLiteralString() {
```

Comment:
In order to reduce the number of changes... why not just change this code to call `_literal.getStringValue()`?

Reply:
because we want caller to explicitly specify what type they want to use. so they have to use getLiteral().getTypeLiteral()